tests/integration/minibroker_test.go

```diff
@@ -145,7 +145,51 @@ var _ = Describe("classes", func() {
 			name:   "postgresql",
 			plan:   "11-7-0",
 			params: map[string]interface{}{},
-			assert: func(instance *apiv1beta1.ServiceInstance, binding *apiv1beta1.ServiceBinding) {},
+			assert: func(instance *apiv1beta1.ServiceInstance, binding *apiv1beta1.ServiceBinding) {
+				By("rendering and loading the postgresql client template")
+				tmplPath := path.Join(testDir, "resources", "postgresql_client.tmpl.yaml")
+				values := map[string]interface{}{
+					"DatabaseVersion": "11.7",
+					"SecretName":      binding.Spec.SecretName,
+					"Command": []string{
+						"sh", "-c",
+						"psql" +
+							" \"host=${DATABASE_HOST} port=${DATABASE_PORT} user=${DATABASE_USER} password=${DATABASE_PASSWORD}\"" +
```

Comment:
We could be using `PGHOST`, `PGPORT`, `PGUSER` and `PGPASSWORD` instead; this line is rather long.  See [documentation](https://www.postgresql.org/docs/9.2/libpq-envars.html).  With that we should be able to drop the use of `sh`.

We weren't able to do it for MySQL because that didn't take _all_ arguments as environment variables.